src/tests/test_graph_web_view.py

```
@@ -0,0 +1,53 @@
+from unittest.mock import MagicMock
+import pytest
```

Flake8 (7.3.0)

[error] 'pytest' imported but unused

(F401)